rpcs3/Input/xinput_pad_handler.cpp

```diff
@@ -419,7 +419,7 @@ PadHandlerBase::connection xinput_pad_handler::update_connection(const std::shar
 		return connection::disconnected;
 
 	dev->state = ERROR_NOT_CONNECTED;
-	dev->state_scp = {};
+	dev->state_scp = {.SCP_ACCEL_Y = -0.21875f};
```

Comment:
what exactly is this value?
And why is it defaulted here

Reply:
Most values in default state in SCP are represented as 0.0, which corresponds to 512 for accelerometers. Y accelerometer in "neutral" state reads as 400 on my pad, which corresponds to -0.21875 SCP.

It's defaulted here so if SCP interface uses unmodified XInput1_3.dll then fields SCP_ACCEL/GYRO won't be changed and "neutral" state is transparently returned for accelerometers.

Reply:
The default is 399 based on all other code.
And you should add a comment for that at least.

I imagine this wouldn't be an issue if this accelerometer support could be checked somehow in your modified version.
You could just skip the assignment if it wasn't supported.